packages/launchpad/src/setup-actions.ts

```diff
@@ -0,0 +1,81 @@
+import { isMobile } from '@automattic/viewport';
+import type { LaunchpadTracksData, Task } from './types';
+
+export const setUpActionsForTasks = (
+	tasks: Task[],
+	siteSlug: string | null,
+	tracksData: LaunchpadTracksData,
+	extraActions?: {
+		setShareSiteModalIsOpen: ( isOpen: boolean ) => void;
```

Comment:
Should `setShareSiteModalIsOpen` also be nullable? I think we can budget on `extraActions` being defined, but `setShareSiteModalIsOpen` not being defined.
```suggestion
		setShareSiteModalIsOpen?: ( isOpen: boolean ) => void;
```